From 905c51b53a028cfb942c7a1578bc4004daa3c76e Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Thu, 25 Oct 2018 15:52:11 +0900
Subject: [PATCH 2/3] docs: correct duplicated labels in APIs reference

There are duplicated labels in REST APIs warned while compiling
document. Fix this warning by correcting the labels.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 docs/guides/spp-ctl/api-reference.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/guides/spp-ctl/api-reference.rst b/docs/guides/spp-ctl/api-reference.rst
index 3c22cb9..70eddd6 100644
--- a/docs/guides/spp-ctl/api-reference.rst
+++ b/docs/guides/spp-ctl/api-reference.rst
@@ -753,7 +753,7 @@ vlan object:
 
 classifier table:
 
-.. _table_spp_ctl_spp_vf_res_vlan:
+.. _table_spp_ctl_spp_vf_res_cls:
 
 .. table:: Vlan objects of getting spp_vf.
 
-- 
2.13.1

